oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+account-payment https://github.com/mamcode/account-payment 12.0-mig-account_cash_invoice
```

Comment:
Let's wait for https://github.com/OCA/account-payment/pull/240 to be finished and merged, otherwise it could be overseed since the changes left aren't very big